techlibs/quicklogic/synth_quicklogic.cc

```diff
@@ -268,7 +278,11 @@ struct SynthQuickLogicPass : public ScriptPass {
         if (check_label("map_cells")) {
 
             std::string techMapArgs = " -map +/quicklogic/" + family + "_cells_map.v";
-            techMapArgs += " -map +/quicklogic/" + family + "_lut_map.v";
+            if(vpr) {
+                techMapArgs += " -D NO_LUT -map +/quicklogic/" + family + "_lut_map.v";
```

Comment:
Is your intention here not to map LUTs at all and make the BLIF file use `.names` directives instead?

Reply:
Yes, that is the intention

Reply:
Ok. But keep in mind that it won't work for PP3 currently as the architecture does not provide sites that VPR could use to pack `.names`. This may be added off course.

Reply:
ok, I added a check to not allow -vpr only for PP3 family.

Reply:
Ok.